testing/kuttl/e2e/pgbackrest-init/01-pgbackrest-connect.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+# When the cluster comes up, only the repo in the 0th position has activated with a backup,
+# so the pgbackrest status should be "mixed"
+- script: CLUSTER=init-pgbackrest $HOME/crunchydata/postgres-operator/testing/kuttl/scripts/pgbackrest-initialization.sh "mixed"
```

Comment:
🤔 What is PWD when Kuttl runs? Is there some way we can make this path relative to something? (The test, the suite, the Git checkout...)